server/constants/constants.go

```diff
@@ -49,6 +49,8 @@ const (
 	FilterCreatedByMe     = "me"
 	FilterCreatedByAnyone = "anyone"
 	FilterAllChannels     = "all_channels"
+	FilterAssignmentGroup = "assignment-group"
```

Comment:
FilterAllChannels have underscore between but FilterAssignmentGroup have - . Can we make this consistent ?

Reply:
it is fixed in next PR
https://github.com/Brightscout/mattermost-plugin-servicenow/blob/421889bcd9479236437f97dab5326423a89e2c1d/server/constants/constants.go#L53